tests/test_initial_conditions.py

```diff
@@ -1,57 +0,0 @@
-# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
```

Comment:
combined with tests/initial_conditions/test_initial_conditions and deleted redundant tests.